### DISCUSSION TITLE
Issues Connecting PS4 v2 Controllers and ESP32 for GameCube: Can ESP8266 Serve as an Alternative?

### DISCUSSION THREAD

**Discussion by julyyy240** [darthcloud/BlueRetro#972] [Q&A]
Hello, my problem is the following: I followed a video to make the internal connection of a YakaraColombia video, and I did the internal installation of the ESP32 for the 4 GameCube controllers (I'm sure I followed the tutorial exactly as it was). When trying to connect my PS4 v2 controllers, I am not successful. I also cannot connect to the ESP32 from my mobile device (using the web config); the device does not appear. Now, if I disconnect the 5V cable from the console to the ESP32 (leaving only the GND cable and the 4 GameCube cables connected), and connect the board to the PC (using another power source), the blue LED blinks, and the board appears in the web config. I can also pair the controller, but I cannot get it to respond on the GameCube. Currently, I have version 1.8.3_HW1 loaded, but I have tried HW2 and other earlier versions with the same result. I would also like to know if it is possible to do the same process with an ESP8266, as I have 2 GameCube controllers but only one ESP32, and I also have an ESP8266. Thank you very much.

**Comment by darthcloud** [comment]
Use GagmeCube HW1 FW and get a real ESP32 module from Espressif.

This project only for original ESP32.

**Reply by julyyy240** [reply]
Hello, I have the ESP32, which is not working for me. The specific model is ESP-32S NodeMCU v1.1, and I am attaching photos of my board. Could you tell me if it is suitable for the BlueRetro for GameCube? As I'm experiencing the issue I've mentioned."
![Imagen de WhatsApp 2024-04-23 a las 16 40 38_12591b41](https://github.com/darthcloud/BlueRetro/assets/90367319/a73b2239-d04b-4580-8f38-10dc44a1726a)
![Imagen de WhatsApp 2024-04-23 a las 16 40 38_1d4ac043](https://github.com/darthcloud/BlueRetro/assets/90367319/3cdc3e38-e504-43fe-84e9-8cf39cbcec4c)
![Imagen de WhatsApp 2024-04-23 a las 16 42 36_044ee041](https://github.com/darthcloud/BlueRetro/assets/90367319/f1f42da8-2b1c-4993-9e93-77e414b60dca)
